src/main/java/jenkins/plugins/git/GitSCMSource.java

```diff
@@ -123,21 +127,59 @@ public String getDisplayName() {
         }
 
         public ListBoxModel doFillCredentialsIdItems(@AncestorInPath SCMSourceOwner context,
```

Comment:
same. Otherwise further changes will be also causing formal compatibility issues